client/packages.go

```diff
@@ -58,6 +58,7 @@ type Snap struct {
 	Broken           string        `json:"broken,omitempty"`
 	Contact          string        `json:"contact"`
 	License          string        `json:"license,omitempty"`
+	CommonIds        []string      `json:"common-ids,omitempty"`
```

Comment:
Should this be at a snap level or at an app level?

Reply:
both :-)